.github/workflows/build_and_test.yml

```diff
@@ -30,7 +30,7 @@ jobs:
         run: npm ci
 
       - name: Build Web App
-        run: npx gulp build --platform=browser
+        run: npx gulp build --platform=browser --flavor=debug
```

Comment:
Maybe just `--debug` or `--release`?

Reply:
That sounds good to me.
At least on Android, the values for flavor dimensions must be unique. So the flavor dimension value can be mapped to the dimension.
In this case, the dimension is "build type", and the values `debug` and `release`. Passing `--debug` is clear that we should set the build type to `debug`.

Reply:
I'm not necessarily sure. If I saw `debug` the first thing I'd assume is that we were setting the log level to `debug` - naming the flag is more explicit and reduces confusion.

Reply:
Good point! I agree.